src/client/extension.ts

```diff
@@ -162,6 +162,13 @@ export async function activate(
             (fl) => ({ ...fl, uri: fl.uri.fsPath } as WorkspaceFolder)
         ) || [];
     
+    const data = getPortalsOrgURLs(workspaceFolders);
+    // Added this loop to get all the orgURLs customers are working on. Combination of orgURLs and vscodemachineId help us determine desktop usage
+    data?.forEach(value =>{
+        // eslint-disable-next-line @typescript-eslint/no-non-null-assertion,  @typescript-eslint/no-extra-non-null-assertion
+        const orgId = value.toLowerCase().match(/([a-z,A-Z,0-9]*.[a-z,A-Z,0-9]*.dynamics.com)/g)![0] // extract the orgURLs from the manifest file name.
```

Comment:
Are Portals available for Sovereign clouds?  If so, this regex will miss the *.crm.dynamics.cn for China, *.crm.microsoftdynamics.us and *.crm.appsplatform.us for UsGov

Reply:
thanks Crash for pointing this. Fixed